lib/index.js

```diff
@@ -2879,6 +2879,7 @@ var Connection = exports.Connection = function (host, port, callback, options, d
     this.error;
     this._max_cached_query = options.maxCachedQuery || -1;
     this._cache_query = options.cacheQuery?{}:null;
+    this._messageFile = options.messageFile || __dirname + "/firebird.msg";
```

Comment:
Can be better to use `path.join(__dirname, 'firebird.msg')` instance of `__dirname + '.firebird.msg'` ?

Reply:
Ops, I'm so sorry... it is really better. I'll fix it hurry

Reply:
Done, thanks @maratth

Reply:
Perfect